content/en/dora_metrics/setup/deployments.md

```diff
@@ -252,7 +252,10 @@ DORA Metrics for the service `shopist` only consider the Git commits that includ
 
 - Change lead time stage breakdown metrics are only available for GitHub.
 - Change lead time is not available for the first deployment of a service that includes Git information.
-- If commits on a feature branch are squashed into a single commit prior to being merged into the default branch, the commit history for that pull request is not included in the change lead time calculation. The first commit included in the calculation is the newly created single commit.
+- For rebased branches, *change lead time* calculations consider the new commits created during the rebase, not the original commits.
+- When using "Squash and merge" for pull requests:
+  - For GitHub: Metrics are emitted for the original commits.
+  - For other git providers: Metrics are emitted for the new commit added to the target branch.
```

Comment:
In other providers, is it also called "Squash and merge"?

Reply:
Great point!

Gitlab:
<img width="605" alt="image" src="https://github.com/user-attachments/assets/604b0412-4147-4860-8438-430dd1705366">

Bitbucket:
<img width="599" alt="image" src="https://github.com/user-attachments/assets/1e87e534-ef86-4b66-939d-21b0e254adc6">

I think we should just say 'Squash' and not Squash and merge